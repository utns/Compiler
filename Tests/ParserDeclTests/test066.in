procedure foo();
const
    a =1;
type
    b = integer;
var
    c: b;
begin
end;
    