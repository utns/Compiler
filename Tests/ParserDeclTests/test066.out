integer                  type 
float                    type 
char                     type 
foo                       proc
args:          
locals:        
     a                   const int               1
     b                  type alias         integer
     c                         var               b
